src/page-content/content-index.tsx

```diff
@@ -52,38 +51,39 @@ export const featureContentCloud: FeatureContentProps = {
   id: 'index.featurecontent.cloud',
   slant: true,
   background: 'grey',
-  overline: <>Ory Cloud</>,
+  overline: <>The future of identity</>,
   title: (
     <>
-      <span className={'is-mute-text'}>Powerful,</span>
-      {''} elegant
+      <span className={'is-mute-text'}>Never build authentication</span>
+      {''} again
     </>
   ),
   description: (
     <>
-      Authenticate and manage users, set and check permissions, protect your
-      APIs, applications, and data - all inside an intuitive console.
+      Ory provides adaptable services and APIs for authentication,
+      authorization, access control, OAuth2.0, Zero Trust.
     </>
   ),
   button: {
-    ctaTitle: 'Sign up',
-    to: 'https://console.ory.sh',
+    ctaTitle: 'Learn more',
+    to: 'https://www.ory.sh/docs/',
     iconRight: ArrowRight
   },
   content: (
     <StaticImage
       loading="lazy"
       className="responsive"
-      alt="Ory Project Dashboard"
-      src="../images/illustrations/dashboard.png"
+      alt="Ory Open Source Ecosystem Illustration"
+      src="../images/illustrations/integration.png"
```

Comment:
![Screenshot 2021-12-02 at 15 39 37](https://user-images.githubusercontent.com/24988314/144443340-1b136f5a-3c53-466b-898f-39e6e6fbda56.png)
```suggestion
      src="../images/illustrations/examplelogin.png"
```